packages/cli/src/reporters/util.ts

```diff
@@ -161,7 +195,7 @@ function formatHttpResponse (response: any) {
 
 function formatLogs (logs: Array<{ level: string, msg: string, time: number }>) {
   return logs.flatMap(({ level, msg, time }) => {
-    const timestamp = DateTime.fromMillis(time).toLocaleString(DateTime.TIME_WITH_SECONDS)
+    const timestamp = DateTime.fromMillis(time).toLocaleString(DateTime.TIME_24_WITH_SECONDS)
```

Comment:
I see no mention of these changes in the PR description, could you add it there as well?

Reply:
fixed